workflow-gateway/templates/service.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
+      targetPort: {{ .Values.appConfig.PORT }}
```

Comment:
you should leave this as http, it doesn't mean 80 here, it is just a label that is referenced to the deployment which has something like this in it:

```
ports:
            - name: http
              containerPort: {{ .Values.appConfig.port }}
              protocol: TCP
```

So it will take 8080 automatically